app/license_test.go

```diff
@@ -37,3 +38,75 @@ func TestRemoveLicense(t *testing.T) {
 		t.Fatal("should have removed license")
 	}
 }
+
+func TestSetLicense(t *testing.T) {
+	th := Setup()
+	defer th.TearDown()
+
+	l1 := &model.License{}
```

Comment:
Worth breaking these out as table driven tests?

Reply:
I just copied / pasting without even looking. But yeah, this would probably be a good candidate for that.